src/services/codefixes/importFixes.ts

```diff
@@ -370,7 +370,7 @@ function createImportAdderWorker(sourceFile: SourceFile | FutureSourceFile, prog
         );
 
         let fix: FixAddNewImport | ImportFixWithModuleSpecifier;
-        if (existingFix && importKind !== ImportKind.Namespace) {
+        if (existingFix && importKind !== ImportKind.Namespace && existingFix.kind !== ImportFixKind.UseNamespace && existingFix.kind !== ImportFixKind.JsdocTypeImport) {
             fix = {
                 ...existingFix,
                 addAsTypeOnly,

```

Comment:
This broke because `fix` is a union of objects, except the two props added here are _not_ actually in two of the fixes, so the code was actually tacking random props onto fixes that would then simply never be read by anyone.